src/index.js

```diff
@@ -0,0 +1,245 @@
+'use strict'
+
+const defaultMigrations = require('../migrations')
+const repoVersion = require('./repo/version')
+const repoLock = require('./repo/lock')
+const repoInit = require('./repo/init')
+const isBrowser = require('./option-node')
+const errors = require('./errors')
```

Comment:
Suggestion: destructure the errors you're using?

Reply:
Well, to be honest, I learned in Python not to do this. First of all, it gives more context about the function/error (not really applicable in this case) and when you add new function/error into the module you don't have to think about adding it in the import. Also when you remove something you don't have to again think about removing it from import...